lua/sim/units/StructureUnit.lua

```diff
@@ -41,8 +41,8 @@ local GetTarmac = import("/lua/tarmacs.lua").GetTarmacType
 
 -- compute once and store as upvalue for performance
 local StructureUnitRotateTowardsEnemiesLand = categories.STRUCTURE + categories.LAND + categories.NAVAL
-local StructureUnitRotateTowardsEnemiesArtillery = categories.ARTILLERY * (categories.TECH2 + categories.TECH3 + categories.EXPERIMENTAL)
-local StructureUnitOnStartBeingBuiltRotateBuildings = categories.STRUCTURE * (categories.DIRECTFIRE + categories.INDIRECTFIRE) * (categories.DEFENSE + (categories.ARTILLERY - (categories.TECH3 + categories.EXPERIMENTAL)))
+local StructureUnitOnStartBeingBuiltRotateBuildings = categories.STRUCTURE * ((categories.DIRECTFIRE + categories.INDIRECTFIRE + categories.ANTINAVY) + (categories.DEFENSE + categories.ARTILLERY))
```

Comment:
```suggestion
local StructureUnitOnStartBeingBuiltRotateBuildings = categories.STRUCTURE * (categories.DIRECTFIRE + categories.INDIRECTFIRE + categories.ANTINAVY + categories.DEFENSE + categories.ARTILLERY)
```

Reply:
I'm not a fan of Volcano rotating.
```suggestion
local StructureUnitOnStartBeingBuiltRotateBuildings = categories.STRUCTURE * ((categories.DIRECTFIRE + categories.INDIRECTFIRE + categories.ANTINAVY) + (categories.DEFENSE - categories.uab4201 + categories.ARTILLERY))
```